tests/rules/PasswordExpiredTest.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * @author Jörn Friedrich Dreyer <jfd@butonic.de>
+ *
+ * @copyright Copyright (c) 2018, ownCloud GmbH
+ * @license GPL-2.0
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\PasswordPolicy\Rules;
```

Comment:
test namespace

Reply:
Tests for a Class should reside in the same namespace, but finish with *Test, see https://github.com/owncloud/core/issues/30739#issue-303801425

Reply:
^a proposed definition which is not yet approved or applied
in contrary we have naming conventions in place which we *try to* follow for years
We can change once decided - until then: please do as we do it now